src/components/Layout.tsx

```diff
@@ -262,15 +291,6 @@ export default ({
   )
 }
 
-export const pageQuery = graphql`
-  fragment site on Site {
-    siteMetadata {
-      title
-      description
-      author {
-        name
-      }
-      keywords
-    }
-  }
-`
+export default Layout;
+
+export const pageQuery = siteFragment;
```

Comment:
`siteFragment` is used in two places:

1. Here to infer the type correctly
2. And for `codegen` to generate fragment correctly (wasn't working for me at the beginning)

Reply:
Ah gotcha! really smart!